### ISSUE TITLE
Schema link needs updating in Example 13

### ISSUE THREAD

**Issue by David-Chadwick** [w3c/vc-data-model#382]
Example 13 (https://w3c.github.io/vc-data-model/#ex-13-usage-of-the-credentialschema-property-to-perform-json-schema-validation) was updated to replace proof of age with university degree, but the reference to the schema was not updated at the same time. This should be changed to

```
},
  "credentialSchema": {
    "id": "https://example.org/universities/degree.json",
    "type": "JsonSchemeDegree"
  },

```
or something similar

**Comment by David-Chadwick** [comment]
p.s. A similar change is needed to Example 14 as well

**Comment by msporny** [comment]
Fixed in f4d9ce8aec62e5c6dd0bf260cd0cc25ba39083b2. Thanks for the catch, @David-Chadwick!
